evm_loader/cli/src/types/tracer_ch_db.rs

```diff
@@ -36,9 +64,22 @@ impl ClickHouseDb {
         }
     }
 
+    pub fn get_block_hash(&self, slot: u64) -> ChResult<String> {
+        block(|| async {
+            let query = "SELECT hash FROM events.notify_block_local where slot = ? ";
+            self.client
+                .query(query)
+                .bind(slot)
+                .fetch_one::<String>()
+                .await
+                .map_err(std::convert::Into::into)
+        })
+    }
+
     pub fn get_block_time(&self, slot: Slot) -> ChResult<UnixTimestamp> {
         block(|| async {
-            let query = "SELECT JSONExtractInt(notify_block_json, 'block_time') FROM events.notify_block_local WHERE (slot = toUInt64(?))";
+            let query =
+                "SELECT JSONExtractInt(notify_block_json, 'block_time') FROM events.notify_block_local WHERE (slot = toUInt64(?))";
```

Comment:
I think we can remove toUint64() here, but we need to check it.

Reply:
removed